src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.hpp

```diff
@@ -37,7 +37,7 @@
                        Register tmp1, Register tmp2,
                        VectorRegister vr1, VectorRegister vr2,
                        VectorRegister vrs,
-                       bool is_latin, Label& DONE, bool is_m2 = true);
+                       bool is_latin, Label& DONE, Assembler::LMUL lmul = Assembler::m2);
```

Comment:
I think it's safer to pass this lmul param explictly at all call site. So no need for a default param.

Reply:
Yes, I was thinking so too.